tsconfig.json

```diff
@@ -5,14 +5,8 @@
     "pretty": true,
     "baseUrl": ".",
     "paths": {
-      "@looker/sdk": ["packages/sdk/src"],
-      "@looker/sdk-codegen-scripts": ["packages/scripts/src"],
-      "@looker/sdk-codegen": ["packages/sdk-codegen/src"],
-      "@looker/sdk-utils": ["packages/utils/src"],
-      "@looker/test-utils": ["packages/test-utils/src"]
+      "@looker/*": ["packages/*/src"]
```

Comment:
Dynamic (read zero-config) package setup for TSconfig as well!

Reply:
Excellent. I was hoping we'd be able to get to that.